content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -246,6 +246,8 @@ be down for the same period as the previous example (`08:29:00` to `10:21:00`,)
 The CronJob is only responsible for creating Jobs that match its schedule, and
 the Job in turn is responsible for the management of the Pods it represents.
 
+Starting with Kubernetes v1.32 the job created by CronJob controller will have an annotation `batch.kubernetes.io/cronjob-scheduled-timestamp` which will show the original (expected) creation timestamp for a Job. The value of this field is in RFC3339 format.
```

Comment:
I think it is a bit confusing to place this after:

```
The CronJob is only responsible for creating Jobs that match its schedule, and
the Job in turn is responsible for the management of the Pods it represents.
```

Can we either:
- highlight here why this annotation is useful? For example: `can be useful if the job has not been created according to the schedule`
- or just place above the last sentence or somewhere else in the Job creation section?

Reply:
Good point, moved it up, after the first paragraph.